example/main.cpp

```diff
@@ -22,8 +24,16 @@ int main(int argc, char *argv[]) {
     spdlog::debug("Inexor vulkan-renderer, BUILD " + std::string(__DATE__) + ", " + __TIME__);
     spdlog::debug("Parsing command line arguments.");
 
-    inexor::vulkan_renderer::Application renderer(argc, argv);
-    renderer.run();
-    renderer.calculate_memory_budget();
+    std::unique_ptr<inexor::vulkan_renderer::Application> renderer;
+
+    try {
+        renderer = std::make_unique<inexor::vulkan_renderer::Application>(argc, argv);
+    } catch (inexor::vulkan_renderer::Exception &exception) {
+        spdlog::critical(exception.what());
+        std::abort();
+    }
+
+    renderer->run();
+    renderer->calculate_memory_budget();
```

Comment:
No catch here?

Reply:
That's not necessary for now as VMA will not throw an exception and there's no other code part in there.